src/frontend/components/Chat/SharedChatComponents/InputDisplayNameOverlay/index.spec.tsx

```diff
@@ -92,7 +98,150 @@ describe('<InputDisplayNameOverlay />', () => {
     expect(mockSetOverlay).not.toHaveBeenCalled();
   });
 
+  it('enters a valid nickname but the server takes too long to answer.', async () => {
+    const mockConverse =
+      converse.claimNewNicknameInChatRoom as jest.MockedFunction<
+        typeof converse.claimNewNicknameInChatRoom
+      >;
```

Comment:
This can be made before the `describe` block. The `afterEach` function is responsible to reset it and you can still mock the implementation in every tests.

Reply:
Yes it was unecessary thanks